src/components/Controls/RangeControl.vue

```diff
@@ -38,7 +38,7 @@ export default {
     },
     step: {
       type: Number,
-      default: 1
+      default: 0.001
```

Comment:
@TimPietrusky Why are we changing this?

Reply:
@2xAA I think I needed a more fine-grained step than just 1, I guess I wanted to see the exact value of the audio-feature already in the range-slider. If it's just 1, then it will not show fine-grained data. I could move this into a new PR if you want.

Reply:
@TimPietrusky No problem, makes sense!